### PR TITLE
Add Instagram logo and variables to website

### PR DESCRIPTION
**Pull request by BenGitsMagic**
Jekyll variables for Youtube and Instagram don't work, not yet figured out.

### CODE REVIEW

File: _pages/contact.html [hunk 1]
```diff
@@ -29,10 +29,10 @@ <h5>Emails</h5>
                 <td>
                     <h5>Social</h5>
                     <ul>
-                        <li><a href="https://facebook.com/robotics1418">Facebook</a></li>
```
Comment (ErikBoesen) [inline _pages/contact.html:32]: Are we not using the facebook page anymore?

**Comment by BenGitsMagic** [comment]
The "update contact.html" was me reordering the links in the contacts page

**Comment by ErikBoesen** [comment]
What issue are you seeing with using the Instagram and YouTube variables? My best guess is that you haven't tried restarting your local Jekyll server. Unlike other changes that will automatically propagate when you save the files, `_config.yml` changes won't be reflected until you stop and restart the `bundle exec jekyll serve` command.

File: _pages/contact.html (hunk 1 above)
Comment (BenGitsMagic) [reply _pages/contact.html:32]: Yep it's pretty dead so Mr. Knight said to remove it

**Comment by BenGitsMagic** [comment]
You're right I restarted my jekyll serve command and it worked so there isn't actually any issue with the links

File: _pages/contact.html (hunk 1 above)
Comment (ErikBoesen) [reply _pages/contact.html:32]: I get removing it from the header, but not listing it at all under contact seems strange

Comment (AndrewLester) [reply _pages/contact.html:32]: Just added @BobbyMi11er to the Facebook as an admin.

**Review by BobbyMi11er — APPROVED** [review]

**Pull request merged**